ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/disconnected/fullyprivate/cco-manual-workload-identity/deprovision/cucushift-installer-rehearse-azure-ipi-disconnected-fullyprivate-cco-manual-workload-identity-deprovision-chain.metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+	"path": "cucushift/installer/rehearse/azure/ipi/disconnected/fullyprivate/cco-manual-workload-identity/deprovision/cucushift-installer-rehearse-azure-ipi-disconnected-fullyprivate-cco-manual-workload-identity-deprovision-chain.yaml",
```

Comment:
I know we recently we hit some issues on azure + oidc + private cluster, so I wonder can we reproduce the issue without `disconnected`?